src/nls/fi/strings.js

```diff
@@ -511,6 +514,7 @@ define({
     "INVALID_VERSION_NUMBER"               : "Paketin versionumero ({0}) on virheellinen.",
     "INVALID_BRACKETS_VERSION"             : "{APP_NAME}in yhteensopivuuden merkkijono ({0}) on virheellinen.",
     "DISALLOWED_WORDS"                     : "Sanat ({1}) eivät ole sallittuja kentässä {0}.",
+    "NPM_INSTALL_FAILED"                   : "npm install -komento epäonnistui: {0}",    
```

Comment:
It makes sense to replace a dash with `\u2011` (non-breaking hyphen) to prevent line break after it.